widget/menu_desktop_test.go

```diff
@@ -44,7 +44,7 @@ func TestMenu_Layout(t *testing.T) {
 		"normal": {
 			windowSize: fyne.NewSize(500, 300),
 			menuPos:    fyne.NewPos(10, 10),
-			wantImage:  "menu_layout_normal.png",
+			wantImage:  "menu/layout_normal.png",
```

Comment:
menu/desktop/layout_normal.png?

Reply:
Unless menu/desktop/layout_normal.png and menu/mobile/layout_normal.png should be the same?

Reply:
Yes, they should. That’s why I put them into `menu/` not `menu/desktop` or `menu/mobile`.

Reply:
I don't understand why some of these menu tests are desktop specific, they don't seem to access desktop features or avoid mobile limitations?
To stuart's comment above if half of this is not a desktop test, perhaps it does not belong in a file ending in `_desktop_test`?